### DISCUSSION TITLE
IPCC Images of burning-ember

### DISCUSSION THREAD

**Discussion by Enakshi-1998** [petermr/petermr#34] [General]
[https://www.google.com/search?q=burning+embers+diagram&client=firefox-b-1-d&sxsrf=ALiCzsaL-9eAGJ9nOD9xe4bqE2Ok0NABTA:1654529006479&source=lnms&tbm=isch&sa=X&ved=2ahUKEwiDmc7tkJn4AhVcbDABHeDQA78Q_AUoAXoECAEQAw&biw=1282&bih=986&dpr=2.4](url)

This link was sent by a member of the meeting (Hackathon) yesterday. They had reproduced some of the figures mentioned in the IPCC reports. Can you have a look at it there?

**Comment by petermr** [comment]
The link doesn't work for me.

**Reply by Enakshi-1998** [reply]
https://www.google.com/search?q=burning+embers+diagram&source=lnms&tbm=isch&sa=X&ved=2ahUKEwir_ou125v4AhXVR2wGHRZACx0Q_AUoAXoECAEQAw&biw=1280&bih=569&dpr=1.5

Can you try this one instead?
